### ISSUE TITLE
m2 error upon first running

### ISSUE THREAD

**Issue by SuperCowboyDinosaur** [jlonge4/local_llama#12]
streamlit.errors.StreamlitAPIException: `set_page_config()` can only be called once per app page, and must be called as the first Streamlit command in your script. 
is thrown after following instructions and filling the env_vars. 
looked at the code and it does seem like you are calling it correctly, at least to my untrained eye, so a bit unsure where that's coming from?

**Comment by jlonge4** [comment]
@SuperCowboyDinosaur hmmm...is it still happening to you?

**Comment by SuperCowboyDinosaur** [comment]
nope. the suggested changes worked for me just fine. did you try? and also, if you did , what error are you getting? still the same? in that case...which os/versions do you have? 
pretty sure i understand where this is coming from so hopefully i can help ?

**Comment by SuperCowboyDinosaur** [comment]
sorry for previous response, different repo...
yes , this still happens on this one for me. still unsure wtf is going on.

**Comment by jlonge4** [comment]
@SuperCowboyDinosaur 

Checkout v3, and follow the new update instructions in the read me. You won't be disappointed.
